<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <arg name="communication_namespace" default="vehicleA"/>

  <!-- <node pkg="communication_gate" type="gate_node" name="gate_node" ns="$(arg communication_namespace)" output="screen"> -->
  <node pkg="communication_gate" type="gate_node" name="gate_node" ns="$(arg communication_namespace)" args="$(arg communication_namespace)" output="screen">
    <remap from="/$(arg communication_namespace)/tx_com" to="/tx_com"/>
    <!-- <remap from="/computation/status" to="computation/status"/> -->
    <!-- <remap from="/cav_vehicle_model/output" to="cav_vehicle_model/output"/> -->
    <!-- <remap from="/lane_waypoints_array" to="/$(arg communication_namespace)/lane_waypoints_array"/> -->
    <!-- <remap from="/computation/cooperation_status" to="/$(arg communication_namespace)/computation/cooperation_status"/> -->
  </node>

  <!--<node pkg="cav_vehicle_model" type="model" name="model" ns="vehicleB" output="screen">
    <param name="L" value='1.2' />
    <param name="m" value='600' />
    <param name="dt" value='0.01' />
    <param name="R" value='0.3' />
    <param name="I_d" value='0.3'/>
    <param name="I_g" value='0.3' />

    <remap from="output" to="output"/>
    <remap from="input" to="input"/>
  </node>
-->

</launch>
